ipie/estimators/handler.py

```diff
@@ -244,3 +244,107 @@ def zero(self):
         self.global_estimates[:] = 0.0
         for _, e in self.items():
             e.zero()
+
+
+class EstimatorHandlerFP(EstimatorHandler):
```

Comment:
Why is this necessary? It looks identical % a different name for Block vs Iter

Reply:
there are some other differences in how things are printed etc. also some walker properties don't make sense for free projection